ibm_catalog.json

```diff
@@ -0,0 +1,311 @@
+              "flavors": [
+                "standard"
+              ],
+              "catalog_id": "f64499c8-eb50-4985-bf91-29f9e605a433",
+              "optional": true,
+              "input_mapping": [
+                {
+                  "dependency_input": "log_analysis_provision",
+                  "version_input": "log_analysis_enabled"
+                },
+                {
+                  "dependency_output": "log_analysis_ingestion_key",
+                  "version_input": "log_analysis_ingestion_key"
+                },
+                {
+                  "dependency_input": "cloud_monitoring_provision",
+                  "version_input": "cloud_monitoring_enabled"
+                },
+                {
+                  "dependency_output": "cloud_monitoring_access_key",
+                  "version_input": "cloud_monitoring_access_key"
+                },
+                {
+                  "dependency_input": "region",
+                  "version_input": "cloud_monitoring_instance_region"
+                }
+              ]
+            }
+          ],
+          "dependency_version_2": true,
+          "configuration": [
+            {
+              "key": "cloud_monitoring_access_key",
+              "required": true
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_name",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_tags",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_tolerations",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_enabled",
+              "required": true
+            },
+            {
+              "key": "cloud_monitoring_endpoint_type",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_instance_region",
+              "display_name": "Region",
+              "required": true,
+              "custom_config": {
+                "type": "vpc_region",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "generationType": "2"
+                }
+              }
+            },
+            {
+              "key": "cloud_monitoring_metrics_filter",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_secret_name",
+              "required": false
+            },
+            {
+              "key": "cluster_config_endpoint_type",
+              "required": false
+            },
+            {
+              "key": "cluster_id",
```

Comment:
this should be a custom dropdown. See https://github.ibm.com/GoldenEye/observability-agents-module/blob/7277a823bd27af328fc0f5a3ef77e0869391781a/ibm_catalog.json#L722-L732